src/axom/inlet/examples/example1_expected_command_line_output

```diff
@@ -0,0 +1,16 @@
+thermal_solver/mesh/filename: found /data/star.mesh

```

Comment:
@white238 -- should these files live in the `axom_data` repo?  https://github.com/LLNL/axom_data

Specifically, the input file, the output table and the `expected_command_line_output` file?
That way, you can add lots of examples without cluttering the inlet `examples` directory.

Reply:
(one possible answer, other than "no", could be "yes, but we'll move it after the code/format settles a bit more")